python/lsst/ts/mtdome/mock_llc/mock_motion/base_llc_motion_without_crawl.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of ts_mtdome.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from abc import ABC
+
+from ...enums import LlcMotionState
+from .base_llc_motion import BaseLlcMotion
+
+
+class BaseLlcMotionWithoutCrawl(BaseLlcMotion, ABC):
```

Comment:
Why the ABC here? Is it because get_position_velocity_and_motion_state is still not defined? Is it necessary?

Reply:
In a related question: would it practical to define get_position_velocity_and_motion_state in the two BaseLlcMotionWith/WithoutCrawl classes, rather than leaving that for more detailed subclasses?

I'm surprised how different the three implementations are, so perhaps not. But I figured I'd ask. I confess some of them are so complicated that it makes me rather nervous -- it looks hard to test all possible cases.

Reply:
`ABC` is there indeed because of `get_position_velocity_and_motion_state`. When I remove `ABC`, PyCharm complains but flake8 and mypy don't so I removed it and the import.

Reply:
Regarding your related question: the methods are quite different for azimuth and elevation. Later on the louvers will be added as well and those will have a very different implementation than the shutter. So I'll leave this as is.